snapcraft/cli/remote.py

```diff
@@ -99,6 +102,11 @@ def remote_build(
         "snapcraft remote-build is experimental and is subject to change - use with caution."
     )
 
+    if debug:
+        logging.getLogger("snapcraft.internal.remote_build._launchpad").setLevel(
```

Comment:
Please live this to the developer env we have or if this is user facing, then let's punt this until we have a better design for general logging wrt verbosity

Reply:
Removed